### ISSUE TITLE
Add command for wolframalpha

### ISSUE THREAD

**Issue by chrisjrob** [chrisjrob/sluggle#12]
Assuming there is an API

**Comment by grifferz** [comment]
If that is not feasible, find does its !units or = command by wrapping GNU units:

```
<grifferz> =2 picograms per gigafortnight as lb per hour
<find> 2 picograms per gigafortnight = 1.3122754e-26 lb per hour
<find> 2 picograms per gigafortnight = (1 / 7.6203518e+25) lb per hour
```

**Comment by chrisjrob** [comment]
Seems that the rather outdated WWW::Wolfram module has the advantage of being "official":

http://blog.wolframalpha.com/2011/04/18/wolframalpha-and-duckduckgo-partner-on-api-binding-and-search-integration/

Looks fairly usable but still on version 1. But then the Wolfram website provides the competing Net::WolframAlpha module for download at http://products.wolframalpha.com/api/libraries.html

I will install the latter via cpan and compare.

**Comment by chrisjrob** [comment]
Could get that to work, so installed the one they provided instead, seems to work fine. The issue now is that it returns a number of alternative responses, for example if you search for a place it will return "pods" for location, population, time, weather, map, regions, cities, airports. Clearly it is impossible to present all that information, and also clearly choosing between them is nigh on impossible.

It could respond with something like:

> <chrisjrob> sluggle: wolfram H�nefoss
> <sluggle> chrisjrob: H�nefoss,  Buskerud, Norway. Options are: 1. Population, 2. Location, 3. Local map, 4. Administrative regions, 5. Current local time, 6. Current weather, 7. Nearby cities, 8. Nearby airport.
> <chrisjrob> sluggle: 1
> <sluggle> chrisjrob: city population | 13930 people  (country rank: 37th)  (2007 estimate)

Sounds fairly involved to code - quite different to the simple command/response so far.

**Comment by chrisjrob** [comment]
I've created a new branch 'wolfram' as this is clearly going to be a longer project.

**Comment by chrisjrob** [comment]
In the end I opted to simply return the first response from Wolfram. The user can then search again with more queries. E.g.

First search "guildford" would return lots of options, but a second search of "population of guildford" should work.

Implemented in commit 3a57b16f4afba1709d8b4aceaef37ae2e85bd4cf
